docs/root/intro/version_history.rst

```diff
@@ -30,6 +30,7 @@ Version history
 * http: mitigated a race condition with the :ref:`delayed_close_timeout<envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.delayed_close_timeout>` where it could trigger while actively flushing a pending write buffer for a downstream connection.
 * http: added support for :ref:`preserve_external_request_id<envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.preserve_external_request_id>` that represents whether the x-request-id should not be reset on edge entry inside mesh
 * http: changed `sendLocalReply` to send percent-encoded `GrpcMessage`.
+* http: added support for rejecting requests containing invalid HTTP header values.
```

Comment:
So if we leave this in, we should reference the config field.  If we runtime guard let's mention the string we guard it with.

Reply:
Oops, I think we now have two release notes - how about removing this one since I think the upper one is correct?

Reply:
actually this file looks odd in general - there seem to be a bunch of deltas which aren't yours.  Bad merge?